pytrip/cube.py

```diff
@@ -289,6 +290,10 @@ def create_empty_cube(self, value, dimx, dimy, dimz, pixel_size, slice_distance)
         self.num_bytes = 2
         self.data_type = "integer"
         self.pydata_type = np.int16
+        self.slice_pos = []
+        for _i, i in enumerate(range(dimz)):
+            self.slice_pos.append(slice_distance * i + slice_offset)
```

Comment:
Why do you need enumerate here ?

The same code can be written in more compact way:
```self.slice_pos = [slice_distance * i + slice_offset for i in range(dimz)]```